src/main/java/com/artipie/auth/YamlPermissions.java

```diff
@@ -50,24 +49,11 @@ public YamlPermissions(final YamlMapping yaml) {
     }
 
     @Override
-    public boolean allowed(final Authentication.User user, final String action) {
+    public boolean allowed(final AuthUser user, final String action) {
```

Comment:
This class should be removed. Isn't it?

Reply:
It will be removed as soon as all adapter will migrate to new permissions model, including docker (and for docker we need to implement its own permissions).